app/controllers/message_objects_controller.rb

```diff
@@ -29,6 +29,17 @@ def download
     send_data @message_object.content, filename: MessageObjectHelper.displayable_name(@message_object), type: @message_object.mimetype, disposition: :download
   end
 
+  def download_pdf
+    authorize @message_object
+
+    pdf_content = @message_object.prepare_pdf_transformation
```

Comment:
Toto mi este znie trosku divne, "pdf transformacia" je predsa xslt a nie ten vysledok ci? nemalo by to by nieco ako `convert_to_pdf` ?

Reply:
Alebo ako pozeram na ten mixin tak mozno `prepare_pdf_visualization`

Reply:
Premenovala som